UPGRADING.md

```diff
@@ -244,3 +245,11 @@ See [anomaly-detection#35](https://github.com/opensearch-project/anomaly-detecti
 | `KIB`    | `OSD`                   |
 
 See [anomaly-detection-dashboards-plugin#1](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/1) for an example.
+
+#### API Compatibility with OpenSearch Plugins
```

Comment:
Super nit, can you add "Rest" to the API to make it consistent.

Reply:
Done